api-spec/pgs_api.yaml

```diff
@@ -20,6 +20,8 @@ paths:
           required: true
           description: PGS-generated GUID of the request to get
           example: 77e1c83b-7bb0-437b-bc50-a7a58e5660ac
+      security:
```

Comment:
since these spec will no longer be used, it is better to remove the file

Reply:
file deleted, but please note that the remote api spec specified in the package.json file are actually not available:
https://raw.githubusercontent.com/pagopa/pagopa-infra/jwt-token-auth-pgs/src/core/api/payment_transactions_gateway_api/pgs-fe/v1/_openapi.json.tpl

Reply:
change the refs in this way https://raw.githubusercontent.com/pagopa/pagopa-infra/main/src/core/api/payment_transactions_gateway_api/pgs-fe/v1/_openapi.json.tpl

Reply:
updated